runtime-testsuite/test/org/antlr/v4/test/runtime/csharp/global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "projects": [
+    "replace_this"
+  ]
+}
+
```

Comment:
Not only useful and also wasted me tons of times to get it right; the current project.json business, although is fading away (but not yet, thus we still need to live with this nonsense for now), can only reference another project in this way, while in csproj file, you can just tell it with a path.